src/core/tests/test_models.py

```diff
@@ -1234,14 +1234,209 @@ def test_account_queryset_update_or_create_with_defaults(self):
             ]
         )
 
-    def test_organization_deduplicate_to_ror_record(self):
-        self.organization_bbk_legacy.deduplicate_to_ror_record()
+class TestOrganizationManagers(TestCase):
```

Comment:
Although not added here, the test `test_ror_validation` is now failing:

`django.db.utils.DataError: value too long for type character varying(10)
`

Reply:
Roger--that was fixed on the test fix branch merged into master a few days ago.